### ISSUE TITLE
Include arm64 builds in CI

### ISSUE THREAD

**Issue by jorgebay** [polarstreams/polar#44]
We should run tests on GitHub actions for arm64.

We could use something like https://github.com/uraimo/run-on-arch-action that uses QEMU and run `go test ./...` or any approach.

**Comment by sunilkumardash9** [comment]
I think this should do the job. It seems to be working
```
run_on_arm64:
      runs-on: ubuntu-latest
#       timeout-minutes: 10
      steps:
       - uses: actions/checkout@v3
         
       - name: Run on aarch64
         uses: uraimo/run-on-arch-action@v2
         with:
          arch: aarch64
          distro: ubuntu_latest
          githubToken: ${{ github.token }}
          base_image: golang:1.19
          install: |
                apt-get update -q -y
                apt-get install -q -y git
          run: |
#          for vcs stamping error
            git config --global --add safe.directory "$PWD"   
            go build ./...
            go test -v ./... 
            go test -v -tags=integration -p=1 ./internal/test/integration
```

**Comment by sunilkumardash9** [comment]
Does it look good?

**Comment by jorgebay** [comment]
(Sorry I missed the notif the first time)

Yeah, looks pretty straightforward, do you want to send a pull request?

**Comment by sunilkumardash9** [comment]
sure, (This time  I missed)

**Comment by mkingori** [comment]
@jorgebay I guess this issue should be closed

**Comment by jorgebay** [comment]
Thats right!
Thanks @mkingori !

Fixed in #184
